### PR TITLE
Smart library demo

### PR DESCRIPTION
**Pull request by FenixFly**
Smart Library Python Demo.

The application is a demo of automated “smart library”.  It involves the registration of the reader; authorization of the reader through face recognition; receiving and returning books by recognizing QR codes generated for each book in the library.

### CODE REVIEW

File: demos/python_demos/smart_library_demo/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+                             [-h_lm LMHEIGHT] [-br BR] [-lib LIB] [-w WEB]
+                             [--no_show]
+
+Smart Library Sample
+
+optional arguments:
+  -h, --help         show this help message and exit
+  -fr FREC           Required. Type of recognizer. Available DNN face
+                     recognizer - DNNfr
+  -m_rd RDMODEL      Required. Path to .xml file
```
Comment (eizamaliev) [inline demos/python_demos/smart_library_demo/README.md]: Please specify which kind of model needs here.
Same for `-m_fd`

File: demos/python_demos/smart_library_demo/smart_library_demo.py [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+import face_recognizer as fr 
+import book_recognizer as br
+
+sys.path.insert(0, 'src/infrastructure')
+from DynamicDatabase import *
+
+uknownID = -1
+count = 0
+
+def CreateArgparse():
```
Comment (eizamaliev) [inline demos/python_demos/smart_library_demo/smart_library_demo.py]: Please, use snake case for function names.

Comment (FenixFly) [reply demos/python_demos/smart_library_demo/smart_library_demo.py]: Done in commit https://github.com/opencv/open_model_zoo/pull/882/commits/98aad7ead4a363071c097ffa225f9267cac64c5d

File: demos/python_demos/smart_library_demo/README.md (hunk 1 above)
Comment (FenixFly) [reply demos/python_demos/smart_library_demo/README.md]: Done in commit https://github.com/opencv/open_model_zoo/pull/882/commits/460d3bb33951bee775dae2fbc0eb7b8bf96214c7

**Pull request closed without merging**